README.md

```diff
@@ -1,27 +1,38 @@
 
 ## Jekyll Installation
-Swap out `rbenv` below for `rvm` if you prefer. RVM was giving me installation issues, so I found `rbenv` - Travis
+Jekyll runs on Ruby, and running multiple versions of Ruby on a single system is apparently difficult enough to support at least two environment managers, `rbenv` and `rvm`, which allow you to select which version of Ruby you'd like to use. We include instructions here for `rbenv`, but feel free to use `rvm` if you'd like.
```

Comment:
_Both_ `rbenv` and `rvm` allow you to choose the version?

Reply:
That's right - they are two tools that do effectively the same thing.